komodo-modeshape-vdb/src/test/resources/config/komodo.cnd

```diff
@@ -0,0 +1,408 @@
+/*
```

Comment:
This is part of komodo-core which has a dependency on komodo-modeshape-vdb. It makes no sense to push it down into here even for test purposes. What is the rationale for it to be here?